tests/contrib/bottle/test.py

```diff
@@ -71,7 +71,7 @@ def hi(name):
         if ddtrace.config.bottle.trace_query_string:
             assert s.get_tag(http.QUERY_STRING) == query_string
         else:
-            assert http.QUERY_STRING not in s.meta
+            assert http.QUERY_STRING not in s.get_tags()
```

Comment:
this is ok, but alternative is:

```suggestion
            assert s.get_tag(http.QUERY_STRING) is None
```